x-pack/plugins/infra/server/lib/adapters/nodes/processors/last/metric_buckets_processor.ts

```diff
@@ -4,18 +4,19 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { set } from 'lodash';
+import { cloneDeep, set } from 'lodash';
 import { InfraESSearchBody, InfraProcesorRequestOptions } from '../../adapter_types';
 import { createBasePath } from '../../lib/create_base_path';
-import { metrics as metricFunctions } from '../../metrics';
+import { metrics as metricAggregationCreators } from '../../metric_aggregation_creators';
```

Comment:
Looks like the files of the renamed metric aggregation creators are missing :wink: This is also what seems to cause the CI failures.